[2022-03-15 19:32:48,308] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [queued]>
[2022-03-15 19:32:48,314] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [queued]>
[2022-03-15 19:32:48,314] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:32:48,314] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-03-15 19:32:48,314] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:32:48,321] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-03-13T09:00:00+00:00
[2022-03-15 19:32:48,323] {standard_task_runner.py:54} INFO - Started process 3064 to run task
[2022-03-15 19:32:48,365] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-03-13T09:00:00+00:00', '--job_id', '77', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp6uptw4a0']
[2022-03-15 19:32:48,365] {standard_task_runner.py:78} INFO - Job 77: Subtask twitter_aluraonline
[2022-03-15 19:32:48,387] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [running]> on host 1a222f561522
[2022-03-15 19:32:48,424] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-03-15 19:32:48,425] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-13T09:00:00.00Z&end_time=2022-03-14T09:00:00.00Z
[2022-03-15 19:32:48,456] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 19:32:48,732] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-13T09:00:00.00Z&end_time=2022-03-14T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqmi34t8iowbeq8l2n9pdi314ltp
[2022-03-15 19:32:48,733] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 19:32:48,955] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220313T090000, start_date=20220315T193248, end_date=20220315T193248
[2022-03-15 19:32:53,303] {local_task_job.py:102} INFO - Task exited with return code 0
[2022-03-15 21:33:32,951] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [queued]>
[2022-03-15 21:33:32,956] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [queued]>
[2022-03-15 21:33:32,957] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:33:32,957] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-03-15 21:33:32,957] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:33:32,963] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2022-03-13T09:00:00+00:00
[2022-03-15 21:33:32,964] {standard_task_runner.py:54} INFO - Started process 9036 to run task
[2022-03-15 21:33:32,995] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2022-03-13T09:00:00+00:00', '--job_id', '78', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpa9yv87b7']
[2022-03-15 21:33:32,996] {standard_task_runner.py:78} INFO - Job 78: Subtask twitter_aluraonline
[2022-03-15 21:33:33,014] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2022-03-13T09:00:00+00:00 [running]> on host 1a222f561522
[2022-03-15 21:33:33,047] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-03-15 21:33:33,048] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-13T09:00:00.00Z&end_time=2022-03-14T09:00:00.00Z
[2022-03-15 21:33:33,069] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 21:33:33,349] {twitter_hook.py:37} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2022-03-13T09:00:00.00Z&end_time=2022-03-14T09:00:00.00Z&next_token=b26v89c19zqg8o3fpyqmi34t8iowbeq8l2n9pdi314ltp
[2022-03-15 21:33:33,350] {logging_mixin.py:112} WARNING - /run/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2022-03-15 21:33:33,554] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20220313T090000, start_date=20220315T213332, end_date=20220315T213333
[2022-03-15 21:33:37,947] {local_task_job.py:102} INFO - Task exited with return code 0
